src/ProductsXmlFeed.php

```diff
@@ -377,4 +401,24 @@ private static function get_taxonomies( $product_id ) {
 
 		return wp_list_pluck( $terms, 'name' );
 	}
+
+	/**
+	 * Fetch shipping object.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return Shipping
+	 */
+	private static function get_shipping() {
+		if ( null === self::$shipping ) {
+			self::$shipping = new Shipping();
+			/**
+			 * When we start generating lets make sure that the cart is loaded.
+			 * Various shipping and tax functions are using elements of cart.
+			 */
+			wc_load_cart();
```

Comment:
🤩  

This fixed the issue with the WC session.

Reply:
👍🏻